projects/plugins/jetpack/extensions/blocks/ai-assistant/transforms/index.tsx

```diff
@@ -7,23 +7,41 @@ import TurndownService from 'turndown';
  * Internal dependencies
  */
 import { blockName } from '..';
-import { EXTENDED_BLOCKS, isPossibleToExtendBlock } from '../extensions/ai-assistant';
+import {
+	EXTENDED_BLOCKS,
+	ExtendedBlockProp,
+	isPossibleToExtendBlock,
+} from '../extensions/ai-assistant';
 /**
  * Types
  */
 import { PromptItemProps } from '../lib/prompt';
 
-const turndownService = new TurndownService( { emDelimiter: '_' } );
+const turndownService = new TurndownService( { emDelimiter: '_', headingStyle: 'atx' } );
```

Comment:
Setting `headingStyle` to `atx` will transform `<h1 />` Text` to `# Text` instead of `Text\n====`

https://github.com/mixmark-io/turndown#options